config.py

```diff
@@ -61,10 +81,19 @@ def getenv(env_var, default=NOT_PROVIDED):
     "development": "stamps-export.csv",
 }[ENVIRONMENT_TYPE]
 
+# list of modules like: "AlignmentNewsletterSearch Eliza Silly Random"
+# if $STAMPY_MODULES = "ALL", enable everything found in ./modules
+enabled_modules_var = getenv_unique_list("STAMPY_MODULES")
+if enabled_modules_var == set(["ALL"]):
+    enabled_modules = All_Stampy_Modules
+    log.info("Loading all modules indiscriminately")
+else:
+    assert (enabled_modules_var != set([NOT_PROVIDED])), "Please choose your modules with $STAMPY_MODULES"
```

Comment:
please support default value (`"ALL"`) instead of error message when not provided - no need for a breaking change that would crash the main stampy if Rob didn't update the env file before merging

Reply:
Ok, will do both. I still feel that having a default-fail is better, as it reports its failure immediately and can be fixed quickly, whereas accidentally loading a module could lead to it doing things on your system or over APIs that you don't want it to do.